library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -443,8 +443,40 @@ public int markRowDeleted(long... ids) {
      * @param ids the IDs of the downloads to remove
      * @return the number of downloads actually removed
      */
-    public int remove(long... ids) {
-        return markRowDeleted(ids);
+    public int removeDownloads(long... ids) {
```

Comment:
Wondering why did you created this method, isn't it the same as `markDownloadRowsForDeletion()`? Or maybe the later should be private rather than exposed.